src/NSwag.Sample.OpenApi/NSwag.Sample.OpenApi.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Now that I look again at the existing samples, this one isn't needed. And, I started with the wrong template here, ignoring the lack of actual APIs exposed and bulking the project up with irrelevant files.